### PR TITLE
VEN-1109 | User profile winter storage

### PR DESCRIPTION
**Pull request by Nizar-Rahme**
## Description :sparkles:
- Make most of the profile berths components, generic
- Use the common components for winter storage views
- Extract all the common
### Closes :no_good_woman:
- [VEN-1109](https://helsinkisolutionoffice.atlassian.net/browse/VEN-1109): Customer winter storage page

### CODE REVIEW

File: src/features/profile/berths/Berths.tsx [hunk 1]
```diff
@@ -1,45 +1,148 @@
-import React from 'react';
+import { useTranslation } from 'react-i18next';
```
Comment (TatuArvela) [inline src/features/profile/berths/Berths.tsx:1]: Shouldn't React be imported here?

File: src/features/profile/components/properties/Properties.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+type CommonProperties = Record<
+  | 'electricity'
+  | 'gate'
+  | 'lighting'
+  | 'wasteCollection'
+  | 'water'
+  | 'summerStorageForDockingEquipment'
+  | 'summerStorageForTrailers',
+  boolean
+>;
```
Comment (TatuArvela) [inline src/features/profile/components/properties/Properties.tsx:13]: This should probably be called `AllProperties`, as all the properties are not common between the two place types

File: src/locales/en.json [hunk 3]
```diff
@@ -26,6 +26,8 @@
     "season": "Season",
     "section": "Section",
     "sent": "Sent",
+    "storage_for_docking_equip": "Winter storage of accessories",
+    "storage_for_trailers": "Summer storage of trailers",
```
Comment (TatuArvela) [inline src/locales/en.json:30]: `trailer_summer_storage` is already in these keys, they should probably be merged?

**Review by TatuArvela — APPROVED** [review]
Left some comments, but altogether it looks good to me 👍

File: src/features/profile/berths/Berths.tsx (hunk 1 above)
Comment (Nizar-Rahme) [reply src/features/profile/berths/Berths.tsx:1]: With the new JSX transformation, importing React is [no longer needed](https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#whats-different-in-the-new-transform).

This is covered by version 16.4 which we're using.
> A version of React that supports the new transform (React 17 RC and higher supports it, but we’ve also released React 16.14.0, React 15.7.0, and React 0.14.10 for people who are still on the older major versions).

As for this PR, I will remove the import statements from the rest of the files.

Comment (TatuArvela) [reply src/features/profile/berths/Berths.tsx:1]: TIL, good job 👍

File: src/features/profile/components/properties/Properties.tsx (hunk 2 above)
Comment (Nizar-Rahme) [reply src/features/profile/components/properties/Properties.tsx]: Fair enough, renamed! 😅

File: src/locales/en.json (hunk 3 above)
Comment (Nizar-Rahme) [reply src/locales/en.json:30]: The Swedish translation seems a bit different, which one do you suggest keeping?
```JSON
    "storage_for_trailers": "Sommarförvaring av trailer",
    "trailer_summer_storage": "Släp sommarförvaring",
```

Comment (TatuArvela) [reply src/locales/en.json:30]: "Släp sommarförvaring" seems to me the more natural one of these two

**Pull request merged**